pkg/oc/cli/startbuild/startbuild_test.go

```diff
@@ -26,7 +26,7 @@ import (
 
 	buildapi "github.com/openshift/origin/pkg/build/apis/build"
 	buildclient "github.com/openshift/origin/pkg/build/client/internalversion"
-	"github.com/openshift/origin/pkg/oauth/generated/clientset/scheme"
+	"github.com/openshift/origin/pkg/oauth/generated/internalclientset/scheme"
```

Comment:
seems like this is moving us backwards.

Reply:
It's not, that package should be removed long time ago. This will be eventually turned to external once @juanvallejo and I get there.